server/server.sh

```
@@ -30,4 +30,12 @@ python3 "$(dirname "$0")/resolve_model_dirs.py" \
     --config "${PADDLEX_HPS_PIPELINE_CONFIG_PATH}" \
     --base-dir "/root/.paddlex/official_models"
 
-exec tritonserver --model-repository="${MODEL_REPO_DIR}" --backend-config=python,shm-default-byte-size=104857600,shm-growth-byte-size=10485760 --log-info=1 --log-warning=1 --log-error=1
+# Start the standalone instance status server in the background
+python3 "$(dirname "$0")/status_server.py" &
+STATUS_SERVER_PID=$!
+
+# Trap to ensure status server is killed when tritonserver exits
+trap "kill $STATUS_SERVER_PID 2>/dev/null" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)